standard/expressions.md

```diff
@@ -1123,9 +1123,13 @@ primary_no_array_creation_expression
     | unchecked_expression
     | default_value_expression
     | anonymous_method_expression
+    | pointer_member_access     // unsafe code support
+    | pointer_element_access    // unsafe code support
     ;
 ```
 
+*pointer_member_access* is available only in unsafe code ([§23.6.3](unsafe-code.md#2363-pointer-member-access)). *pointer_element_access* is available only in unsafe code ([§23.6.4](unsafe-code.md#2364-pointer-element-access)).
```

Comment:
To avoid repetition maybe this could be:

> *pointer_member_access* ([§23.6.3](unsafe-code.md#2363-pointer-member-access)) and *pointer_element_access* ([§23.6.4](unsafe-code.md#2364-pointer-element-access)) are only available in unsafe code ([§23](unsafe-code.md#23-unsafe-code)).

(It is technically correct as is, this is just wordsmithing.)